modules/tensor_mechanics/doc/content/modules/tensor_mechanics/NewBackground.md

```diff
@@ -0,0 +1,139 @@
+exactly the same results.  There is no difference in the final results
+when using a [updated Lagrangian](kernels/lagrangian/UpdatedLagrangianStressDivergence.md) or
+[total Lagrangian](kernels/lagrangian/TotalLagrangianStressDivergence.md) model.*
+
+However, at times one formulation may be more convenient than another.
+For example, the [homogenization system](tensor_mechanics/Homogenization.md)
+system only works with the [total Lagrangian kernel](kernels/lagrangian/TotalLagrangianStressDivergence.md)
+because of the difficulty in including the extra homogenization
+field in the kinematic spatial derivatives.
+In theory it can be more efficient to couple a "native" Cauchy stress
+constitutive model to the updated Lagrangian configuration and a
+"native" 1st Piola-Kirchhoff model to the total Lagrangian configuration.
+However, the currently-implemented
+[material system](tensor_mechanics/NewMaterialSystem.md) always coverts
+Cauchy stress to 1st Piola-Kirchhoff stress and vice-versa so that models
+can be used with either the updated or total Lagrangian kernels.
+
+Both of these models degenerate to the same formulation for small kinematic
+theory:
+\begin{equation}
+      \begin{align*}
+      s_{ij,j}+b_{i} & = & 0 & \, \mathrm{{on}\, \Omega}\\
+      s_{ij}n_{j} & = & \, \hat{t}_{i} & \mathrm{on}\, \partial\Omega_{n}\\
+      u_{i} & = & \hat{u}_{i} & \, \mathrm{on}\, \partial\Omega_{e}
+      \end{align*}
+\end{equation}
+where $s_{ij}$ is the small (engineering) stress and there is now
+no difference between the current and reference configurations.
+Both the updated and total kernels both both large and small deformation
+theories.  The user can switch between them by setting the
+`large_kinematics` flag to `true` for large deformation theory and
+`false` for small deformation theory.  This flag applies to the kernels,
+the [strain calculator](materials/lagrangian/ComputeLagrangianStrain.md), the
+[homogenization system](/tensor_mechanics/Homogenization.md), and
+for many models, the [constiutive](tensor_mechanics/NewMaterialSystem.md) models for calculating the stress.
+ As such, it is often convenient to set it in the `GlobalParams` section
+ of the input file so that you can easily switch between the two.
+
+## The `use_displaced_mesh` Flag
+
+In MOOSE the `use_displaced_mesh` flag indicates whether gradients
+and integrals are with respect to the current (`true`) or reference (`false`)
+configurations. 
+In the old [StressDivergenceTensors](/StressDivergenceTensors.md) it can
+be confusing to determine when to use the flag.  With the new Lagrangian
+kernels it is simple: the only object that needs to have `use_displaced_mesh`
+set is the [updated Lagrangian](kernels/lagrangian/UpdatedLagrangianStressDivergence.md)
+when `large_kinematics = true`.  The kernel enforces this condition with 
+an error.  For all other objects in the new mechanics system the
+flag should be set to `false`.
```

Comment:
The existing system works the same way. You only set `use_displaced_mesh=true` in the Kernel, and it should not be set for anything else. To avoid confusion, this should be re-worded to say that this is the same as the existing system. You also need to set it for other kernels like Gravity, which would still be the case.

Reply:
That's a good point, they are doing the same thing. I just removed the reference to the current kernels and cleaned up the resulting sentence.